src/main/resources/db/migration/V202409111604__add_column_ownerId_to_pet_table.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE pet
+ADD COLUMN owner_id BIGINT;
```

Comment:
Please remove from pr, not relevant for this pr.
Add it to another pr and I will review.